examples/ellipsometry/eln_data.yaml

```diff
@@ -58,9 +58,6 @@ colnames:
 - Delta
 - err.Psi
 - err.Delta
-definition: NXellipsometry
```

Comment:
why is it needed to be moved to the code and not to keep it separate in the config file?